user-guide/Advanced_Modules/EPM/Tutorials/EPM_Introduction_Tutorial.md

```diff
@@ -0,0 +1,192 @@
+---
+uid: EPM_Introduction_Tutorial
+---
+
+# Getting started with EPM
+
+This tutorial will provide a brief introduction on how you can use many EPM-specific features.
+
+Expected duration: 30 minutes
+
+> [!NOTE]
+> The content and screenshots for this tutorial have been created using DataMiner version 10.4.5.
+
+## Prerequisites
+
+- A DataMiner System that is [connected to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud), and where the [CPEIntegration](xref:Overview_of_Soft_Launch_Options#cpeintegration) soft-launch option is enabled.
+
+> [!TIP]
+> A [DataMiner as a Service](xref:Creating_a_DMS_on_dataminer_services) system comes with a dataminer.services connection out of the box and has most soft-launch options enabled, so it automatically meets this requirement.
+
+## Overview
+
+This tutorial consists of the following steps:
+
+- [Step 1: Deploy the DataMiner EPM Integration Training package from the Catalog](#step-1-deploy-the-dataminer-epm-integration-training-package-from-the-catalog)
+- [Step 2: Create a new collector element and rebalance the system](#step-2-create-a-new-collector-element-and-rebalance-the-system)
+- [Step 3: Use EPM relations to filter out alarms](#step-3-use-epm-relations-to-filter-out-alarms)
+- [Step 4: Using the EPM Feed in Dashboards and Low-Code Apps](#step-4-using-the-epm-feed-in-dashboards-and-low-code-apps)
+- [Step 5: Link EPM entities to Views](#step-5-link-epm-entities-to-views)
+
+## Step 1: Deploy the *DataMiner EPM Integration Training* package from the Catalog
+
+1. Go to <https://catalog.dataminer.services/details/b661f936-d6e7-447c-baee-f0a5503e75b4>.
+
+1. Deploy the Catalog item to your DataMiner Agent by clicking the *Deploy* button.
+
+   > [!TIP]
+   > See also: [Deploying a Catalog item to your system](xref:Deploying_a_catalog_item)
+
+1. Open DataMiner Cube and check whether the following items have been added to your DataMiner Agent:
+
+   - A **view** named *EPM Training*.
+
+   - A **front-end element** named *EPM Training FE*.
+
+   - Two **back-end elements** named *EPM Training BE 1* and *EPM Training BE 2*.
+
+   - Three **collector elements** named *EPM Training Collector 1*, *EPM Training Collector 2*, and *EPM Training Collector 3*.
+
+   - The **Topology app** in between the Surveyor and Activity module.
+
+   ![Cube sidebar with Topology app](~/user-guide/images/EPM_GS_step_1.png)
+
+> [!NOTE]
+> If the Topology app does not show up right away, restart the Cube client.
+
+## Step 2: Create a new collector element and rebalance the system
+
```

Comment:
I'd like to add a small intro at the top of this step to make it more clear why we are doing this. Could you give me some info as to why the user should start this tutorial by adding a collector element?

Reply:
Adding the collector element to the system is to demonstrate how the EPM architecture has horizontal scaling, where a user can add more collectors to reduce the load on individual elements without adding load to the Frontend